src/loaders/svg/tvgSvgSceneBuilder.cpp

```diff
@@ -283,6 +283,27 @@ static Paint* _applyComposition(SvgLoaderData& loaderData, Paint* paint, const S
 }
 
 
+static Paint* _applyFilter(SvgLoaderData& loaderData, Paint* paint, const SvgNode* node, const Box& vBox, const string& svgPath)
+{
+    auto filter = node->style->filter.node;
+    if (!filter || filter->child.count == 0) return paint;
+
+    Scene* scene = Scene::gen();
+    scene->push(paint);
+
+    auto child = filter->child.data;
+    for (uint32_t i = 0; i < filter->child.count; ++i, ++child) {
+        if ((*child)->type == SvgNodeType::GaussianBlur) {
+            auto& gauss = (*child)->node.gaussianBlur;
+            auto direction = gauss.stdDevX > 0.0f ? (gauss.stdDevY > 0.0f ? 0 : 1) : (gauss.stdDevY > 0.0f ? 2 : -1);
+            if (direction == -1) continue;
+            scene->push(SceneEffect::GaussianBlur, direction == 2 ? gauss.stdDevY : gauss.stdDevX, direction, gauss.edgeModeWrap);
```

Comment:
@mgrudzinska could you please confirm here multiple gaussian filters are valid input?
![image](https://github.com/user-attachments/assets/f58f5383-5780-48ce-9062-4d64ef0ee800)

Reply:
please pass the quality value as well. The crash is due to it.
`scene->push(SceneEffect::GaussianBlur, direction == 2 ? gauss.stdDevY : gauss.stdDevX, direction, gauss.edgeModeWrap, 100);`

Reply:
> @mgrudzinska could you please confirm here multiple gaussian filters are valid input? ![image](https://private-user-images.githubusercontent.com/3711518/388368246-f58f5383-5780-48ce-9062-4d64ef0ee800.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzIyMzE0NTAsIm5iZiI6MTczMjIzMTE1MCwicGF0aCI6Ii8zNzExNTE4LzM4ODM2ODI0Ni1mNThmNTM4My01NzgwLTQ4Y2UtOTA2Mi00ZDY0ZWYwZWU4MDAucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTEyMSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDExMjFUMjMxOTEwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NDBhYWQyZDAzN2I4NjRlMjg4OTVjZjMzOTA3YTNjNzQzMTk2OTlkNzMyYjhiNTY1YTZjMDIwYzUxMzFmZDJhMCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.mAV5UBWfH5kT55yTkyu6W9mrViHrvTlJLodcBNtklqw)

yes, the filter contains two gaussian filters, as it is in the svg file - so it is as expected. 
for now we do not support 'in' and 'result' attribs, so they are applied to the whole scene.